crates/nu-cli/src/completions/dotnu_completions.rs

```diff
@@ -35,13 +35,23 @@ impl Completer for DotNuCompletion {
         let prefix_str = prefix_str.replace('`', "");
         let mut search_dirs: Vec<PathBuf> = vec![];
 
-        // If prefix_str is only a word we want to search in the current dir
-        let (base, partial) = prefix_str
-            .rsplit_once(is_separator)
-            .unwrap_or((".", &prefix_str));
+        let (base, partial) = if prefix_str == "~" {
+            ("~", "")
+        } else if let Some((parent, remain)) = prefix_str.split_once(is_separator) {
+            // If prefix_str is only a word we want to search in the current dir.
+            // "/xx" should be split to "/" and "xx".
+            if parent.is_empty() {
+                ("/", remain)
```

Comment:
Probably need to take windows into account. We could use some test cases, the windows version ci workflow will verify them.

Reply:
Oh, and I replaced the `rsplit` with `split` by accident.

Reply:
@hongquan would you please test my change in the other PR to see if it suits your need?